README.aspnet.md

```diff
@@ -1,3 +1,9 @@
+## Important: Client Firewall Rules Update to Microsoft Container Registry (MCR)
+
+To provide a consistent FQDNs, on **March 3, 2020** the data endpoint will be changing from *.cdn.mscr.io to *.data.microsoft.com. 
```

Comment:
There is still a trailing space at the end of this line.

Reply:
fixed